package.json

```diff
@@ -10,6 +10,8 @@
     "start": "webpack --mode development",
     "test": "echo \"No test yet\"",
     "dev": "webpack-dev-server --mode development --open",
+    "sass": "node-sass --include-path scss public/scss/main.scss public/css/style.css",
```

Comment:
The point of adding sass preprocessing is so that webpack can handle the processing of css. we should not be using the `node-sass` package. If webpack is properly configured we can directly use sass in our components by simply importing the sass files like this `import styles from '../../styles.scss'` and access our css classes directly on our components.

**Conclusion**: We do not need the `sass` and `watch-sass` commands you have added, webpack can take care of that. You should look at using the webpack `sass-loader` and enabling **css modules** on webpack.

Reply:
ok